### ISSUE TITLE
Xunit console runner complaining about xunit.core version

### ISSUE THREAD

**Issue by dazinator** [appveyor/ci#2313]
I have a test project with these dependencies:

```csharp
 <ItemGroup>
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.0" />
    <PackageReference Include="Microsoft.AspNetCore.Testing" Version="2.0.0" />
    <PackageReference Include="xunit" Version="2.3.0" />
    <PackageReference Include="Microsoft.Extensions.FileProviders.Composite" Version="1.0.1" />
    <PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" Version="1.0.1" />
    <PackageReference Include="Microsoft.AspNetCore.NodeServices" Version="1.0.0-beta-000007" />
    <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="1.0.2" />
    <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="1.0.2" />
  </ItemGroup>
```

Appveyor automatically discovers the tests and runs them with the xunit console runner but throws this error: `System.IO.FileLoadException: Could not load file or assembly 'xunit.core, Version=2.3.0.3683, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c' or one of its dependencies. `

Here is the log:

> Time Elapsed 00:00:03.32
Discovering tests...OK
%xunit20%\xunit.console.x86 "C:\projects\dazinator-aspnet-extensions-fileproviders\src\Dazinator.AspNet.Extensions.FileProviders.Tests\bin\Debug\net46\Dazinator.AspNet.Extensions.FileProviders.Tests.dll" -appveyor
xUnit.net Console Runner (32-bit .NET 4.0.30319.42000)
  Discovering: Dazinator.AspNet.Extensions.FileProviders.Tests
  Discovered:  Dazinator.AspNet.Extensions.FileProviders.Tests
  Starting:    Dazinator.AspNet.Extensions.FileProviders.Tests
  Finished:    Dazinator.AspNet.Extensions.FileProviders.Tests
=== TEST EXECUTION SUMMARY ===
   Dazinator.AspNet.Extensions.FileProviders.Tests  Total: 44, Errors: 0, Failed: 0, Skipped: 0, Time: 2.376s
%xunit20%\xunit.console.x86 "C:\projects\dazinator-aspnet-extensions-fileproviders\src\Dazinator.AspNet.Extensions.FileProviders.Tests\bin\Debug\net46\Microsoft.AspNetCore.Testing.dll" -appveyor
xUnit.net Console Runner (32-bit .NET 4.0.30319.42000)
System.IO.FileLoadException: Could not load file or assembly 'xunit.core, Version=2.3.0.3683, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)

What is the solution?

**Comment by IlyaFinkelshteyn** [comment]
Seems like you already fixed it, at least while looking at [build history](https://ci.appveyor.com/project/dazinator/dazinator-aspnet-extensions-fileproviders/history). Let us know if you still need help.

**Comment by dazinator** [comment]
Yeah the issue seems to be appveyor auto attempts to run xunit tests built targeting .net framework with the xunit console, and fails in my case.

I made two changes:

1. Switched tests project to target netcoreapp2.0 instead of net framework.
2. Added global cli tool package reference for the dotnet xunit global tool.

This is what the xunit docs show as standard setup for netcoreapp.

However in this case it was acceptable for me to switch over to netcoreapp. If I wanted my tests to stay targeting full net framework I would still be experiencing this problem.
